bhlib/src/main/ets/components/MainPage/BusinessDayAndTime.ets

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright (C) 2022 Application Library Engineering Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { BusinessModel } from './BusinessModel';
+
+@CustomDialog
+export struct TimeSelectorDialog {
+  time: string
+  where: string
+  dayInd: number
+  format: number
+  callback: (selIndex, where, dayInd) => void
+  arr12: string [] = [
```

Comment:
add these arr12 & arr24 variables in separate constant file

Reply:
done